suzieq/config/etc/suzieq-cfg.yml

```diff
@@ -40,8 +40,6 @@ poller:
   #
   # manager:
   #   workers: 1  # Number of workers polling the nodes
-  #   inventory-path: suzieq/.poller/intentory/static_inventory
-  #   inventory-file-name: static_inv
```

Comment:
Is this just removed from the example OR is this truly gone?

Reply:
We do not support these options anymore

Reply:
This is truly gone. I forgot to delete this part from the configuration file while removing the parameter